.travis.yml

```diff
@@ -2,9 +2,12 @@ sudo: false
 
 language: go
 
+go_import_path: github.com/prometheus/graphite_exporter
+
 go:
-- 1.5.4
-- 1.6.2
+ - 1.7.x
```

Comment:
I think 1.9.x would be enough, we don't build graphite-exporter with older golang anyway.

Reply:
Hmm, maybe I should update the circle.yml file also? I just noticed it has 1.6-base as a test image

Reply:
Yup, please do update it.

Reply:
done